templates/test/result.html.ep

```diff
@@ -20,12 +20,14 @@
 <div class="row" id="result-row">
     <div class="col-sm-12">
         %= include 'test/infopanel', worker => $assigned_worker
-        <div role="tabpanel">
+        <div role="tabpanel" style="height: 100%">
```

Comment:
very likely we want to cover all these "height: 100%" in CSS and I am open for suggestions how to achieve that most efficiently as we would need that also on all child elements.

Reply:
I'm afraid there's no simple CSS rule which wouldn't wrongly affect other elements or requires IDs everywhere. The style attribute is at least simple to follow.

Reply:
ok so we keep the style I understand. Fine with me